build-support/generate-bitcodes.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# generate-bitcodes.sh
+#
+# Generate bytecode handlers bitcode file, if necessary.
+# If the file is already present in the provided directory,
+# this script is a no-op.
```

Comment:
I think this changes the semantics of what currently happens.
Previously, as long as you rebuild `noisepage_static`, the bytecode handlers would get updated.
I think we still want that old behavior -- some of the oddest bugs are from having an outdated/incompatible bytecode handler.

I think keeping the old behavior would just be updating the comment + deleting the if test below.
But I may be missing some reasons for your change.

Reply:
Agreed, the cost of re-compiling / re-optimizing is so low compared to a recompilation of the static lib anyway that any savings likely aren't even worth it. I've removed the test so now the script simply performs the three steps necessary to generate the optimized file. I think it still might be worth keeping it as a separate script though because:

- Slightly easier to grok when its in a bash script rather than the `CMakeLists.txt`, and more expressive than just having the free-floating `COMMAND` statements
- More extensible if something changes in the future

If you disagree with the above though I can just move the functionality back to the `CMakeLists.txt`.

Reply:
Nah, sounds fine to me. Thanks!